From b65152ebc03832972115e6d98e50cb6190d01793 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ond=C5=99ej=20Lyson=C4=9Bk?= <olysonek@redhat.com>
Date: Mon, 3 Feb 2020 13:18:13 +0100
Subject: [PATCH 1/3] Drop superfluous global variable definitions

The file minicom.c, by including the minicom.h header, already defines
the global variables 'dial_user' and 'dial_pass'. The object file
minicom.o is always linked to dial.o. Thus the definitions in dial.c
can be dropped.

This fixes linking with gcc 10 which uses -fno-common by default,
disallowing multiple global variable definitions.
---
 src/dial.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/dial.c b/src/dial.c
index eada5ee..d9d481f 100644
--- a/src/dial.c
+++ b/src/dial.c
@@ -146,8 +146,6 @@ static int newtype;
 /* Access to ".dialdir" denied? */
 static int dendd = 0;
 static char *tagged;
-char *dial_user;
-char *dial_pass;
 
 /* Change the baud rate.  Treat all characters in the given array as if
  * they were key presses within the comm parameters dialog (C-A P) and
-- 
2.24.1

